### PR TITLE
ATLAS-184: Clean up database scripts

### PR DESCRIPTION
**Pull request by HelioStrike**
JIRA issue: https://issues.openmrs.org/browse/ATLAS-184

### CODE REVIEW

**Comment by HelioStrike** [comment]
I've been accidentally closing PRs a lot lately :|

File: db/03_demo_data.sql [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+UNLOCK TABLES;
+
+
+--
+-- Dumping data for table `migrations`
+--
+
+LOCK TABLES `migrations` WRITE;
+/*!40000 ALTER TABLE `migrations` DISABLE KEYS */;
+INSERT INTO `migrations` VALUES ('2014_05_20_085005_create_atlas',1),('2014_05_20_093740_create_auth',1),('2014_05_23_205713_update_archive',1),('2014_05_24_082932_change_column_size',1),('2014_05_24_104445_create_admin',1),('2014_05_28_060735_rename_created_by',1),('2014_06_23_213056_add_show_counts',1),('2014_07_09_165540_drop_archive_datechanged_column',1),('2014_07_15_060605_add_uuid_and_action_to_archive',1),('2014_07_31_065930_add_openmrs_version_column_in_atlas',1),('2016_01_22_153112_create_distributions',1),('2016_02_17_170405_insert_distributions',1);
```
Comment (bmamlin) [inline db/03_demo_data.sql]: Are we using this table? If not, then perhaps we should just drop it.

File: db/03_demo_data.sql [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+UNLOCK TABLES;
+
+--
+-- Dumping data for table `types`
+--
+
+LOCK TABLES `versions` WRITE;
+/*!40000 ALTER TABLE `versions` DISABLE KEYS */;
+INSERT INTO `versions` VALUES (1,'2.1'),
+(2,'2.1-SNAPSHOT'),
```
Comment (bmamlin) [inline db/03_demo_data.sql]: This should be a list of OpenMRS versions, not Atlas module versions. I would suggest values like 1.9, 1.10, 1.11, 2.0, 2.1, 2.2, 2.3, 2.4, 2.5, 2.6, 2.7, 2.8, and 2.9.

**Comment by HelioStrike** [comment]
@bmamlin Should we get rid of 'on update CURRENT_TIMESTAMP' on 'date_changed' in atlas table? We're updating the column by supplying it with 'new Date()' each time, so we aren't taking advantage of it, and it may be safer to eliminate it.

**Comment by HelioStrike** [comment]
@bmamlin I think I put all the database changes in updates. Here's a list of updates:

- Remove 'on update CURRENT_TIMESTAMP' on 'date_changed' in atlas table
- Changing image from varchar to mediumblob on atlas and archive tables
- Add 'ON DELETE CASCASE' to auth
- Drop atlas_version column from atlas and archive
- Create rss table

Is there anything I'm missing?

**Pull request merged**